askama_parser/src/node.rs

```diff
@@ -753,7 +753,7 @@ impl From<char> for Whitespace {
 }
 
 #[derive(Debug, PartialEq)]
-pub struct Loop<'a> {
+pub struct NodeLoop<'a> {
```

Comment:
Not a fan of this. I suppose you think conflicts will become an issue because you want to give the same treatment to `Expr`? I feel like maybe we should leave all the single-value `Expr` variants alone, adding wrappers there seems mostly pointless to me -- even here I'm not really convinced about the `Extends` case.

Instead, we could make `expr` and `node` public modules and then callers can reference `expr::Macro` and `node::Macro`, which seems nicer to me.

Reply:
I was inspired by [syn](https://docs.rs/syn/latest/syn/), which does it this way. Yes, you're right, I'll make the module pub.